includes/admin/helpers/class-inbox-api.php

```
@@ -0,0 +1,62 @@
+}
+
+/**
+ * Handles the Inbox API.
+ *
+ * @since x.x
+ */
+class WPBDP_Inbox_API extends WPBDP_Modules_API {
+
+	use WPBDP_Who;
+
+	public function __construct( $license = null ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$license'. (Unused Code Rules)

(UnusedFormalParameter)